eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,91 @@
+$Language = "swift"
+$LanguageDisplayName = "Swift"
+$PackageRepository = "CocoaPods"
+$packagePattern = "*.podspec.json"
+
+function ComputeCocoaPodsSpecUrl($PackageId, $PackageVersion)
+{
+    # The CocoaPods spec repo is a repository for JSONified versions of the PodSpec file. The path to the JSON
+    # file is derived from the name of the package where the repo URL relative URL is:
+    #
+    #   Spec/x/y/z/[Package]/[Version]/[Package].podspec.json
+    #
+    # The x/y/z values are the first four characters from the MD5 hash of the package name. For example 4/4/a
+    # for the package AzureCore.
+
+    $csp = New-Object -TypeName System.Security.Cryptography.MD5CryptoServiceProvider
+    $encoder = New-Object -TypeName System.Text.UTF8Encoding
+    $hash = [System.BitConverter]::ToString($csp.ComputeHash($encoder.GetBytes($PackageId)))
+    $hash = $hash.ToLower() -replace '-', ''
+    $url = "https://raw.githubusercontent.com/CocoaPods/Specs/blob/master/Specs/$($hash[0])/$($hash[1])/$($hash[2])/$PackageId/$PackageVersion/$PackageId.podspec.json"
+    return $url
+}
+
+# Returns the maven (really sonatype) publish status of a package id and version.
+function IsCocoaPodsPackageVersionPublished($PackageId, $PackageVersion)
+{
+    $url = ComputeCocoaPodsSpecUrl -PackageId $PackageId -PackageVersion $PackageVersion
+
+    Write-Host "Checking $url"
+
+    try
+    {
+        $uri = "https://oss.sonatype.org/content/repositories/releases/$groupId/$pkgId/$pkgVersion/$pkgId-$pkgVersion.pom"
```

Comment:
This looks like android not ios.

Reply:
Whoops. Not sure what happened there.

Reply:
Oh OK, I define the URl earlier and didn't remove this (and made a request against the wrong URL ... thanks for picking it up. I'll end-to-end test again).